src/api/shop/entity.ts

```diff
@@ -283,3 +283,37 @@ export interface OrderableShopsParams {
 }
 
 export type OrderableShopsFilter = 'IS_OPEN' | 'DELIVERY_AVAILABLE' | 'TAKEOUT_AVAILABLE' | 'FREE_DELIVERY_TIP';
+
+export interface StoreListParams {
+  sorter?: string;
+  filter?: string | undefined;
```

Comment:
```suggestion
  filter?: string ;
```
undefined를 따로 명시해주지 않아도 괜찮을 것 같아요